### PR TITLE
Add Privacy Manifest

### PR DESCRIPTION
**Pull request by tommyming**
As required by Apple, reachability needs to add privacy manifest to the source code.

If there are any problems please feel free to review, thanks.

Related Issue: #400

### CODE REVIEW

**Comment by tommyming** [comment]
fix #400

File: Sources/PrivacyInfo.xcprivacy [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSPrivacyAccessedAPITypes</key>
+	<array>
+		<dict/>
+	</array>
```
Comment (dusi) [inline Sources/PrivacyInfo.xcprivacy:8]: Could this be simplified to an empty array like so `<array/>`?

Comment (tommyming) [reply Sources/PrivacyInfo.xcprivacy:8]: Suppose this file is generated from Xcode, maybe I just accidentally press the "add" button to create a new dict, let me have a check.

Comment (tommyming) [reply Sources/PrivacyInfo.xcprivacy]: Should be fixed, please review once you are free, thanks a lot @dusi !

**Comment by tommyming** [comment]
@ashleymills would you please add a version tag about this update when you are free? Thanks.

**Comment by corentios** [comment]
Hello, a release is planned before spring ?

**Comment by biscat** [comment]
Hello, is there any plan to put up the latest release for Cocoapods ?

**Pull request merged**